builder/role_image_test.go

```diff
@@ -441,11 +441,11 @@ func TestBuildRoleImages(t *testing.T) {
 	// Check that making the first wait for the second job works
 	secondJobReady := make(chan struct{})
 	mockBuilder.callback = func(name string) error {
-		if strings.Contains(name, "-myrole:") {
+		if strings.Contains(name, "-myrole-") {
 			<-secondJobReady
 			return nil
 		}
-		if strings.Contains(name, "-foorole:") {
+		if strings.Contains(name, "-foorole-") {
```

Comment:
Using `:` will potentially result in Docker not being able to create the underlying volumes. This is the reason why we disallowed the usage of `:`.